mon-pix/app/components/routes/campaigns/restricted/join-sup.js

```diff
@@ -93,7 +95,7 @@ export default class JoinSup extends Component {
 
       schoolingRegistration = this.store.createRecord('schooling-registration-user-association', {
         id: this.args.campaignCode + '_' + this.lastName,
-        studentNumber: this.studentNumber,
+        studentNumber: this.noStudentNumber ? null : this.studentNumber,
```

Comment:
On passe déjà this.studentNumber à `null` à la ligne 142. On pourrait pas passer directement `this.studentNumber` ici ?

Reply:
done